### PR TITLE
xdgdefault plugin: add dmenu support

### PR DESCRIPTION
**Pull request by N-R-K**

### CODE REVIEW

**Comment by jarun** [comment]
CI breaks. Please fix that.

Also, dmenu needs X. Please use a flag like `GUI` (see `nuke`) to enable it.

File: plugins/xdgdefault [hunk 1]
```diff
@@ -36,7 +45,7 @@ app=$(find "$@" -iname '*.desktop' -exec grep '^Name=' {} + \
     | sed -E 's;.+/(.+desktop):Name=(.+);\2:\1;' \
     | sort -t ':' -k 1,1 \
     | column -t -s ':' -o "$(printf '\t')" \
-    | fzf -e --tiebreak=begin \
+    | $menu \
```
Comment (jarun) [inline plugins/xdgdefault:48]: Can you confirm you have executed this with `fzf` and everything works fine?

Comment (jarun) [reply plugins/xdgdefault:48]: @luukvbaal @KlzXS please review once.

**Review by jarun — CHANGES_REQUESTED** [review]

**Comment by jarun** [comment]
Also please update the requirement in the plugin README to `xdg-utils, fzf/dmenu`

File: plugins/xdgdefault (hunk 1 above)
Comment (N-R-K) [reply plugins/xdgdefault:48]: > Can you confirm you have executed this with `fzf` and everything works fine?

Yes, I have, with both fzf and dmenu.

Comment (jarun) [reply plugins/xdgdefault:48]: Thanks for confirming!

**Comment by jarun** [comment]
Thank you!

**Pull request merged**